tests/integration/tier0/test-check-rollback-handling/test_check_rollback_handling.py

```diff
@@ -0,0 +1,116 @@
+import platform
+
+from envparse import env
+
+
+plt = platform.platform()
```

Comment:
It's nitch picking but I would rename it to something that would represent the value inside, Eg. something like `system_release`